x-pack/test/functional/services/ml/lens_visualizations.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+// import { MlDashboardJobSelectionTable } from './dashboard_job_selection_table';
```

Comment:
Did you want to keep the commented out code in this file?

Reply:
Removed in https://github.com/elastic/kibana/pull/133899/commits/0d27e13d6d19a671db0f9b1a7b92441328ffb23a